css/index.css

```diff
@@ -101,17 +167,127 @@ p {
     justify-content: space-evenly;
 }
 
-.middle-content .boxes .box {
+.middle-content .boxes .box1 {
     width: 12.5%;
     height: 100px;
-    background: black;
+    background: teal;
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
+    
 }
 
```

Comment:
Since .box is a parent of each individual box it will inherit the CSS of the parent.  So the only thing that needs to be changed in the individual boxes is the color of the box.